app/command.go

```diff
@@ -615,21 +620,36 @@ func (a *App) CreateCommand(cmd *model.Command) (*model.Command, *model.AppError
 		}
 	}
 
-	return a.Srv().Store.Command().Save(cmd)
+	command, nErr := a.Srv().Store.Command().Save(cmd)
+	if nErr != nil {
+		var appErr *model.AppError
+		switch {
+		case errors.As(nErr, &appErr):
+			return nil, appErr
+		default:
+			return nil, model.NewAppError("CreateCommand", "app.command.createcommand.internal_error", nil, nErr.Error(), http.StatusInternalServerError)
+		}
+	}
+
+	return command, nil
 }
 
 func (a *App) GetCommand(commandId string) (*model.Command, *model.AppError) {
 	if !*a.Config().ServiceSettings.EnableCommands {
 		return nil, model.NewAppError("GetCommand", "api.command.disabled.app_error", nil, "", http.StatusNotImplemented)
 	}
 
-	cmd, err := a.Srv().Store.Command().Get(commandId)
+	command, err := a.Srv().Store.Command().Get(commandId)
```

Comment:
Let's keep this as `cmd` to avoid any unncessary changes. Go also prefers short names, so `cmd` is preferred.